x-pack/platform/plugins/shared/alerting/common/routes/rule/response/schemas/v1.ts

```diff
@@ -475,6 +475,12 @@ export const alertDelaySchema = schema.object(
   }
 );
 
+export const dashboardSchema = schema.arrayOf(schema.object({ id: schema.string() }));
+
+export const artifactSchema = schema.object({
```

Comment:
Same schema is defined in server>application>[rule_schemas ](https://github.com/elastic/kibana/pull/216292/files#diff-19b19826913ccad617e1a12cd27531edd7fe2068ba51532ebfc8fd5dda064ae4). Is there any reason why we have the schemas duplicated in these two files? I checked existing schemas and they are duplicated in these 2 files, `x-pack/platform/plugins/shared/alerting/common/routes/rule/response/schemas/v1.ts` and `x-pack/platform/plugins/shared/alerting/server/application/rule/schemas/rule_schemas.ts`

Can I reuse the schemas from common folder?